src/modules/tccpp/components/starboard.ts

```diff
@@ -2,16 +2,16 @@ import * as Discord from "discord.js";
 import * as mongo from "mongodb";
 
 import { strict as assert } from "assert";
-import { unwrap } from "../utils/misc.js";
-import { EMOJIREGEX, departialize } from "../utils/discord.js";
-import { KeyedMutexSet } from "../utils/containers.js";
-import { M } from "../utils/debugging-and-logging.js";
-import { DAY, MINUTE } from "../common.js";
-import { BotComponent } from "../bot-component.js";
-import { CommandSetBuilder } from "../command-abstractions/command-set-builder.js";
-import { Wheatley } from "../wheatley.js";
-import { EarlyReplyMode, TextBasedCommandBuilder } from "../command-abstractions/text-based-command-builder.js";
-import { TextBasedCommand } from "../command-abstractions/text-based-command.js";
+import { unwrap } from "../../../utils/misc.js";
```

Comment:
Should starboard stuff be in the tccpp modules? Might GP want that some day?

Reply:
I would say we can deal with making it a proper core component when we get there. In its current form, it is tied to the TCCPP channel structure, so I thought easier to just keep it in the tccpp module for now.